packages/framework-integration-tests/src/read-models/cart-read-model.ts

```diff
@@ -38,14 +38,17 @@ export class CartReadModel {
     return this.checks
   }
 
+  @CalculatedField(['cartItems'])
```

Comment:
From a UX point of view (and also in case we need to expand the customization properties in the future) I'd say it is a bit better to make the user write `dependesOn` here. If not, it is not clear what you are defining here. What I mean is this:
```suggestion
  @CalculatedField({dependsOn: ['cartItems']})
```
This way when the user sees this, they don't even need to go to the documentation to understand it 😃

Reply:
Suggestion applied in commit 2077b2c.